Modix.Bot/Modules/PingModule.cs

```diff
@@ -86,46 +106,67 @@ public async Task Ping()
             });
         }
 
-        private string FormatLatency(double latency)
+        private string FormatLatency(double latency, string icon = "📶")
         {
-            if (latency == 0)
+            if (latency == -1)
             {
-                return "📶 Error ⚠️";
+                return $"{icon} Error ⚠️";
             }
 
             var suffix = latency switch
             {
-                _ when latency > 250 => "💔",
+                _ when latency > 300 => "💔",
                 _ when latency > 100 => "💛", //Yellow heart - trust me
                 _ when latency < 100 => "💚", //Green heart - trust me again
                 _ => "❓"
             };
 
-            return $"📶 {latency: 0}ms {suffix}";
+            return $"{icon} {latency: 0}ms {suffix}";
+        }
+
+        private string FormatAvailability(bool isUp)
+        {
+            return $"🌐 {(isUp ? "Up" : "Down")} {(isUp ? "💚" : "💔")}";
         }
 
         private Color LatencyColor(double avg)
             => avg switch
             {
-                _ when avg > 250 || avg < 0 => Color.Red,
+                _ when avg > 300 || avg < 0 => Color.Red,
                 _ when avg > 100 => Color.Gold,
                 _ when avg < 100 => Color.Green,
                 _ => Color.Default
```

Comment:
Same as above comment - this looks like it'll use the default color if the average is 100ms.